src/pyhf/parameters/utils.py

```diff
@@ -27,6 +28,7 @@ def reduce_paramsets_requirements(paramsets_requirements, paramsets_user_configs
         combined_paramset = {}
         for k in paramset_keys:
             for paramset_requirement in paramset_requirements:
+                # undefined: the modifier does not support configuring that property
```

Comment:
what does this comment refer to?

Reply:
just a reminder of what 'undefined' meant in this context... I forgot. It'll help when going through this code again in the future.

Reply:
ok